content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -170,6 +170,33 @@ kubelet instance. A new kubelet instance deletes all the existing Unix sockets u
 `/var/lib/kubelet/device-plugins` when it starts. A device plugin can monitor the deletion
 of its Unix socket and re-register itself upon such an event.
 
+### Device plugin and unhealthy devices
+
+There are cases when devices fail or being shut down. Device Plugin responsibility
+in this case is to notify kubelet about this situation using the `ListAndWatchResponse` stream.
```

Comment:
What do you mean by stream in this case?

If that is the GRPC call, I would maybe just drop "stream".
```suggestion
in this case is to notify kubelet about this situation using the `ListAndWatchResponse` API.
```